modules/authentication-remove-kubeadmin.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-authentication.adoc
+
+[id="removing-kubeadmin-{context}"]
+= Removing the kubeadmin user
+
+Once an identity provider has been defined it is recommended to remove
+the `kubeadmin` user for security reasons.
+
+WARNING
+====
+If this procedure is followed before another user is a `cluster-admin`,
+then {product-title} must be reinstalled. It is not possible to undo
+this command once executed.
+====
+
+.Prerequisites
+
+* You must have configured at least one identity provider.
+* You must have added the `cluster-admin` role to a user.
+* You must be logged in as an administrator.
+
+.Procedure 
+
+* Remove the `kubeadmin` secrets.
```

Comment:
I don't think we have a set standard for this, but I've seen s/secrets./secrets: often.

Reply:
I've adjusted this to use the semicolon.